coralproject/cay
# fixes for Create Search page and User Detail

## What does this PR do?

fixes #596 and #597 
## How do I test this PR?
- open the Saved Searches page
- click the nav for Create Search. (there should be no red screen)
- click a User detail (there should be comments listed instead of a red screen)

@coralproject/frontend
